src/views/GraphView/components/GraphComponent/GraphComponent.tsx

```diff
@@ -238,7 +256,7 @@ function GraphComponent(props: GraphComponentProps) {
    * @param {object} graphObjects graphObjects attached to simulation
    * @param {object} pMap propsMap keeps track of node/link properties and associated values
    */
-  const addGraphObject = useCallback((type, graphData, prop1, prop2, graphObjs, pMap) => {
+  const addGraphObject = useCallback((type: 'link' | 'node', graphData: GeneralRecordType, prop1, prop2, graphObjs: InitialGraphDataType['graphObjects'], pMap: PropsMap) => {
```

Comment:
not a blocker, but these type definitions are getting quite long, it might be worth it to multi-line it

Reply:
I was avoiding swapping to multi lines to keep the diff more readable. this repo has eslint line length rule turned off but maybe in a future PR that should be turned on and all the violations fixed